webpush/push.go

```diff
@@ -89,5 +108,5 @@ func Send(client *http.Client, sub *Subscription, message, token string) (*http.
 
 // A helper for creating the value part of the HTTP encryption headers
 func headerField(headerType string, value []byte) string {
-	return fmt.Sprintf(`%s=%s`, headerType, base64.URLEncoding.EncodeToString(value))
+	return fmt.Sprintf(`%s=%s`, headerType, base64.RawURLEncoding.EncodeToString(value))
```

Comment:
`URLEncoding` and `RawURLEncoding` give padded and non-padded `base64url` encoded strings respectively. Vapid apparently accepts only non-padded `base64url` strings.